src/JanusGraph.Net/IO/GraphSON/GraphSONReaderBuilder.cs

```diff
@@ -0,0 +1,72 @@
+﻿#region License
+
+/*
+ * Copyright 2018 JanusGraph.Net Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#endregion
+
+using System.Collections.Generic;
+using Gremlin.Net.Structure.IO.GraphSON;
+
+namespace JanusGraph.Net.IO.GraphSON
+{
+    /// <summary>
+    ///     Creates a <see cref="GraphSONReader" /> with the default JanusGraph deserializers and allows adding of custom
+    ///     deserializers.
+    /// </summary>
+    public class GraphSONReaderBuilder
```

Comment:
The name doesn't make it clean that it contain janusgraph changes.

Reply:
Ok, I'll prefix it with _Janus_, so it becomes: _JanusGraphSONReaderBuilder_. (Using _JanusGraph_ would result in _JanusGraphGraphSONReaderBuilder_ which looks odd with the duplicate _Graph_.)